modules/ROOT/pages/fix-catalog-info-conformance-issues.adoc

```diff
@@ -0,0 +1,43 @@
+= Fixing API Catalog Information in Exchange
+
+You can check for API catalog information conformance using a ruleset provided by MuleSoft or a custom ruleset. API catalog information is associated with an API asset in Exchange in the asset portal. 
+
+A ruleset provided by MuleSoft that validates API catalog information is:
+
+* API Catalog Information Best Practices
+
+[[view-api-catalog-issues-in-report]]
+== View API Catalog Conformance Issue Details in the Governance Report
+
+To view API catalog conformance issues in the governance validation report:
+
+. In the API Governance console, select the *Governed APIs* tab.
+. If the API has a *Nonconformant* badge, click *View Report* for information about the conformance issue.
+.. View all issues:
++
+Click *View Details* for a ruleset to which the API is nonconformant. 
+.. View issues for the API catalog only:
+... Select *Catalog Info* in the Conformance Breakdown section of the report.
+... Click *View Details* for a ruleset to which the API catalog is nonconformant.
+
+[[fix-api-catalog-issues-in-exchange]]
+== Fix API Catalog Conformance Issues in Exchange
+
+//Update the following to reuse this from Exchange doc 
+
+Fix API Catalog information conformance issues as follows:
+
+. In Exchange, navigate to the API that has catalog information conformance issues.
+
+. Click the edit pen icon in the asset portal.
++
+You can change the asset icon, name, description, contact name, and contact email.
+
+.. To change the icon, click the edit pen icon, select an image file, and then click *Open*.
```

Comment:
isn't this documented in excahnge? or shouldn't it be, and we should just point to that? LGTM, but I think we shouldn't be documenting this

Reply:
Yes, I was torn about it. I was thinking about reusing the doc from Exchange (putting it in a partial), but if you think it's better just to link to it, I'll do that. That's what we do now for the other aspects, and I was considering doing partials for all of them. I agree that in general it's better not to include doc about other products in API Governance, but it does make it harder for customers to follow when they link out to other doc. 
API Catalog Info isn't something obvious, and we don't call it that anywhere in the Exchange doc, so I was trying to compensate for that. I've talked to Jenny about it and we do have the term in the Glossary. Something to pursue later. :)

Reply:
I removed the procedure and added the link instead per your feedback. Thank you!